int8 id
float32 angle_fdb
float32 rpm_fdb
float32 torque_fdb
